FrontEnd/composables/useUser.js

```diff
@@ -1,76 +1,75 @@
 // composables/useAuth.js
-// import { useAuthStore } from '@/store/auth';
-import { useUserStore } from "~/stores/useUserStore";
-import { useNuxtApp } from "#app";
-import { srvLogin, srvSignup } from "~/services/authServices";
-import { srvChangePropic, srvGetWhoToFollow, srvHandleFollow, srvHandleProfile } from "~/services/userServices";
+import { userStore } from "~/stores/userStore";
+import {srvGetWhoToFollow,srvHandleFollow,srvHandleProfile,} from "~/services/api/userServices"
+
 
 export const useUser = () => {
-  const { $api } = useNuxtApp();
-  //   const authStore = useAuthStore();
-  const userStore = useUserStore();
+  const { $customApi } = useNuxtApp();
+  const { $toast } = useNuxtApp();
+
 
-//   const handleAuthError = (error) => {
-//     console.error("Authentication error: ", error.message);
-//   };
 
   const whoTofollow = async () => {
     try {
-        const response = await srvGetWhoToFollow($api);
-        console.log("response after fetching data : whoTofollow", response.data);
-        // postStore.setPosts(response);
-        return response.data
+        const response = await srvGetWhoToFollow()
+        console.log("response after fetching data whoTofollow : ", response);
+        return response
     } catch (error) {
      console.log(error);
+    $toast.error(error?.message ||  'failed to load follow suggestions');
+
     }
   };
 
   const handleFollowUnfollow = async (userId)=>{
     console.log("userId 11 : ",  userId);
     
     try {
-        const response = await srvHandleFollow($api,userId);
-        console.log("response after fetching data handleFollowUnfollow: ", response.data);
-        // postStore.setPosts(response);
-        return response.data
+        const response = await srvHandleFollow({userId})
```

Comment:
srvHandleFollow - Again, not good naming for the function. Functions should describe the actions performed inside. think of something better.

Reply:
I have updated all the services functions with proper naming conventions,please review the latest commit for changes.